lib/autoupdate/agent/process.go

```diff
@@ -312,6 +316,21 @@ func (s SystemdService) IsEnabled(ctx context.Context) (bool, error) {
 	return false, nil
 }
 
+// IsPresent returns true if the service exists.
+func (s SystemdService) IsPresent(ctx context.Context) (bool, error) {
+	if err := s.checkSystem(ctx); err != nil {
+		return false, trace.Wrap(err)
+	}
+	code := s.systemctl(ctx, slog.LevelDebug, "list-unit-files", "--quiet", s.ServiceName)
+	switch {
+	case code < 0:
+		return false, trace.Errorf("unable to determine if systemd service %s is present", s.ServiceName)
+	case code == 0:
+		return true, nil
+	}
+	return false, nil
```

Comment:
```suggestion
	if code < 0 {
		return false, trace.Errorf("unable to determine if systemd service %s is present", s.ServiceName)
		}
	return code==0, nil
```

Reply:
All of the other calls to `systemctl()` use this switch structure, so I wanted to keep it the same for all invocations. I'll take this suggestion for `IsPresent` since it's short, but leave as-is for other functions with a terminal call to `systemctl`.